README.md

```diff
@@ -1 +1,167 @@
-# Hello_Microverse_Project
\ No newline at end of file
+<a name="My Portfolio"></a>
+
+<div align="center">
+  <img src="Albert" alt="logo" width="140"  height="auto" />
```

Comment:
- [ ] This logo image is not showing up because it doesnt have a valid path is src. kindly consider removing it or placing here a valid logo image.